tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -14,12 +14,16 @@ public sealed class NCronJobIntegrationTests : JobIntegrationBase
     [Fact]
     public async Task CronJobThatIsScheduledEveryMinuteShouldBeExecuted()
     {
-        var fakeTimer = TimeProviderFactory.GetTimeProvider();
+        var fakeTimer = new FakeTimeProvider();
         ServiceCollection.AddSingleton<TimeProvider>(fakeTimer);
         ServiceCollection.AddNCronJob(n => n.AddJob<SimpleJob>(p => p.WithCronExpression("* * * * *")));
         var provider = CreateServiceProvider();
 
-        await provider.GetRequiredService<IHostedService>().StartAsync(CancellationToken);
+        // there may be more than one IHostedService, we want to specifically start the CronScheduler
```

Comment:
Well in a unit test not really I'd argue

Reply:
I meant that the library may end up with more than one IHostedService. It's easy to imagine you can have another one to handle background queueing or pre-initialization of anything else you want to run out-of-band. Anyhow it's not necessary at the moment so I'll remove it.